packages/router/package.json

```diff
@@ -104,7 +104,9 @@
     "test:e2e:native": "node e2e/runner.mjs --env chrome",
     "test:e2e:ci": "node e2e/runner.mjs --env chrome-headless --retries 2",
     "test:e2e:bs": "node e2e/runner.mjs --local -e android5 --tag browserstack",
-    "test:e2e:bs-test": "node e2e/runner.mjs --local --env browserstack.local_chrome --tag browserstack"
+    "test:e2e:bs-test": "node e2e/runner.mjs --local --env browserstack.local_chrome --tag browserstack",
+    "prepack": "cp ../../README.md ./README.md",
+    "postpack": "rm ./README.md"
```

Comment:
I don't think using `cp` and `rm` is portable. The package `rimraf` is already included, which is a portable version of `rm`. In the past I've used the `copyfiles` package as a portable replacement for `cp`.